.env.development

```diff
@@ -20,6 +20,10 @@ REACT_APP_SQUARE_LOCATION_ID=E4R1NCMHG7B2Y
 REACT_APP_THINK_CHINATOWN_APPLICATION_ID=sandbox-sq0idb-j_jFtQUzesCshLEjBd-Lxg
 REACT_APP_THINK_CHINATOWN_LOCATION_ID=LP8KMVMD1BFPK
 
+# Square publishable keys for Apex
+REACT_APP_Apex_APPLICATION_ID=sq0idp-ayb-cNC1zt49cYM7MZ6nYQ
+REACT_APP_Apex_LOCATION_ID=LJQ6T0REE7VS6
+
```

Comment:
Are there alternate credentials to use for development?

Reply:
Yeah you can create them via sendchinatownlove@gmail.com. DM us in #engineering